src/sql/workbench/parts/notebook/browser/notebook.component.html

```diff
@@ -9,11 +9,35 @@
 	</div>
 	<div #container class="scrollable" style="flex: 1 1 auto; position: relative; outline: none" (click)="unselectActiveCell()" (scroll)="scrollHandler($event)">
 		<loading-spinner [loading]="isLoading"></loading-spinner>
-		<div class="notebook-cell" *ngFor="let cell of cells" (click)="selectCell(cell, $event)" [class.active]="cell.active">
-			<code-cell-component *ngIf="cell.cellType === 'code'" [cellModel]="cell" [model]="model" [activeCellId]="activeCellId">
-			</code-cell-component>
-			<text-cell-component *ngIf="cell.cellType === 'markdown'" [cellModel]="cell" [model]="model" [activeCellId]="activeCellId">
-			</text-cell-component>
+		<div class="hoverButtonsContainer" *ngIf="(cells && cells.length > 0) && !isLoading">
+			<span class="containerBackground"></span>
+			<button class="hoverButton" (click)="addCell('code', 0)">
+				<div class="addCodeIcon"></div>
+				<span>Add code</span>
```

Comment:
These strings need to be localized. Please provide the string as a getter in the code behind and use `{{addCodeMsg}}` or similar to include them

Reply:
Fixed